lettuce/features/softwaresupport/bind10.py

```diff
@@ -16,13 +16,13 @@
 # Author: Wlodzimierz Wencel
 
 #
-# This file contain management functions for BIND 10 
-# 
+# This file contain management functions for BIND 10
```

Comment:
Huh? Why do we have a bind10 file? This is a forbidden name. Saying it out loud brings bad luck. Remove the whole file. Better yet, burn it with fire :)